hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -623,6 +639,35 @@ public boolean syncDataFromOM() {
     return true;
   }
 
+  private void printFileAndKeyTableCount() throws IOException {
```

Comment:
This method has different handling for `fileTable` and `keyTable`, the reason for this is not obvious. We return if `keyTable` is null even if `fileTable` might not be. This is making business logic assumptions for OM? Why not write this in a manner that does not assume which table can or cannot be `null`.

Reply:
Oh, thanks for pointing out @kerneltime , I think I was debugging something and missed to handle null for both. I have corrected now. Thanks.

Reply:
@kerneltime pls re-review.